### PR TITLE
Add new directives to cluster submission page

### PR DESCRIPTION
**Pull request by kidrahahjo**
## Description
Newly added directives were updated in the API docs but not in the cluster submission page, this PR adds all supported directives to the cluster submission page.

## Checklist:

- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac-docs/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac-docs/blob/master/ContributorAgreement.md).
- [x] My code follows the [code style guideline](https://github.com/glotzerlab/signac-docs/blob/master/CONTRIBUTING.md#code-style) of this project.

### CODE REVIEW

**Comment by klywang** [comment]
Looks good. Just out of curiosity though -- why was the `ngpu` moved in the edit?

File: docs/source/cluster_submission.rst
Comment (vyasr) [inline docs/source/cluster_submission.rst:149]: Is there a reason that you moved `executable` down?

**Review by vyasr — APPROVED** [review]
LGTM, thanks! I had one question but it's not a blocker. @klywang feel free to merge when you've reviewed and are happy.

**Comment by vyasr** [comment]
@klywang @kidrahahjo I didn't see Kelly's review when I reviewed. It seems that both of our questions are related to ordering. Perhaps just alphabetize the glossary and then we can call it good?

**Review by klywang — APPROVED** [review]

**Comment by klywang** [comment]
@kidrahahjo Would you want to alphabetize the glossary before I merge this?

**Comment by kidrahahjo** [comment]
Sorry, I missed the comments. The reason I changed the order is to maintain consistency between the order mentioned in the [API reference docs](https://docs.signac.io/projects/flow/en/latest/api.html#flow-directives) and this.

I will alphabetize the glossary.

**Review by vyasr — CHANGES_REQUESTED** [review]
@kidrahahjo that's a good point, but I think the solution there is to also resort the API docs page that you linked. Could you do that as well so that the two lists match?

**Comment by kidrahahjo** [comment]
@vyasr yes, I'll do it today. I believe it is not a blocker for this PR to get merged?

File: docs/source/cluster_submission.rst
Comment (bdice) [inline docs/source/cluster_submission.rst:134]:
```suggestion
      The fork directive can be set to True to enforce that a particular operation is always executed within a subprocess and not within the Python interpreter's process even if there are no other reasons that would prevent that.
```

**Comment by bdice** [comment]
@kidrahahjo @vyasr @klywang I created a PR to re-sort the directives list. I think this PR should be good to go once reviewers have approved. https://github.com/glotzerlab/signac-flow/pull/535

**Review by vyasr — APPROVED** [review]

**Pull request merged**
